src/opcua-connector/opcua.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  AttributeIds,
+  ClientSession,
+  ClientSubscription,
+  DataValue,
+  MonitoringMode,
+  MonitoringParametersOptions,
+  TimestampsToReturn,
+  Variant,
+} from 'node-opcua'
+import { OpcuaStreams, Settings, Variables } from './types'
+import { fromEvent, Observable } from 'rxjs'
+import { filter, map } from 'rxjs/operators'
+
+export const subscribeValue = async (
+  session: ClientSession,
+  nodeId: string,
+  samplingInterval: number,
+): Promise<[ClientSubscription, Observable<Variant>]> => {
+  const nodeIdSub = {
+    nodeId,
+    attributeId: AttributeIds.Value,
+  }
+  const subscription = await session.createSubscription2({
```

Comment:
No configuration here? Is there a reason for that? I think we made all other connection params configurable in the demo box.

Reply:
we just have 4 parameters optional. url, username, password, pollInterval. The MVP could life with some stupid default settings. :see_no_evil: